src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -1079,6 +1095,10 @@ public InMemoryOutput downloadOutputs(RemoteAction action, RemoteActionResult re
     tmpOutErr.clearOut();
     tmpOutErr.clearErr();
 
+    if (!forcedDownloads.isEmpty()) {
```

Comment:
Consider moving this into `else` block below to make it clear that this is only available in BwoB.

Reply:
Done.